docs/containerization-design-doc.md

```diff
@@ -0,0 +1,384 @@
+# Azkaban Containerized Executions - Design Doc
```

Comment:
Thanks for working on this! 
All the content inside the `docs` folder is Azkaban's official documentation and is live in https://azkaban.readthedocs.io/en/latest/#. Could you please convert this MD file into a RST one which is the format being used? 
Also, please consider structuring containerization related documentation in a TOC way with a high-level/overview page which links together separate subpages/.rst files covering detailed explanation of each individual topic like dispatch, flow/log management, etc.

Reply:
This is a really good point. Will certainly do the conversion to rst and repost this PR.